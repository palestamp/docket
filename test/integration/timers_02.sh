rm ~/.docket_timer_test
set -o verbose
docket timer new work
grep -Ev 'timing' ~/.docket_timer_test
docket timer new work_child --parent work
docket timer new work_child_child --parent work_child
docket timer set work parent work_child_child
grep -Ev 'timing' ~/.docket_timer_test
