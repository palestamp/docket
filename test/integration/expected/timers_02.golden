docket timer new work
grep -Ev 'timing' ~/.docket_timer_test
docket:timer:1:name=*
docket:timer:1:type=abstract
docket:timer:2:name=work
docket:timer:2:type=concrete
docket:timer:2:parent=*
docket timer new work_child --parent work
docket timer new work_child_child --parent work_child
docket timer set work parent work_child_child
error: Cycle found
grep -Ev 'timing' ~/.docket_timer_test
docket:timer:1:name=*
docket:timer:1:type=abstract
docket:timer:2:name=work
docket:timer:2:type=concrete
docket:timer:2:parent=*
docket:timer:3:name=work_child
docket:timer:3:type=concrete
docket:timer:3:parent=work
docket:timer:4:name=work_child_child
docket:timer:4:type=concrete
docket:timer:4:parent=work_child
